mkt/api/paginator.py

```diff
@@ -60,61 +60,55 @@ def page(self, number):
         return page
 
 
-class MetaSerializer(serializers.Serializer):
+class CustomPagination(pagination.LimitOffsetPagination):
     """
-    Serializer for the 'meta' dict holding pagination info that allows to stay
-    backwards-compatible with the way tastypie does pagination (using offsets
-    instead of page numbers), while still using a "standard" Paginator class.
+    Paginator for DRF API views.
```

Comment:
I would have kept the part of the comment explaining that it's using offsets for backwards-compatibility